proseco/core.py

```diff
@@ -46,6 +46,53 @@ def to_python(val):
     return val
 
 
+class ACABox:
+    def __init__(self, size=None):
+        """
+        Class to represent a box half width amplitude.  Can be initialized with
+        either a two-element sequence (y, z) or a scalar (which applies
+        to both y and z.
+
+        :param size: scalar or 2-element sequence (y, z), (arcsec, default=20)
+
+        """
+        if isinstance(size, ACABox):
+            self.y = size.y
+            self.z = size.z
+            return
+
+        try:
+            assert len(size) == 2
+        except TypeError:
+            # len(scalar) raises TypeError
+            if size is None:
+                size = 20
+            self.y = size
+            self.z = size
+        except AssertionError:
+            # Has a length but it is not 2
+            raise ValueError('size arg must be either a scalar or a two-element sequence')
+        else:
+            self.y = size[0]
+            self.z = size[1]
+
+    @property
+    def row(self):
+        return self.y / 5
```

Comment:
This does still use 5.0 as the pixel scale.  As discussed previously this seems fine to me and makes things tidy for acq where I want 20 pixels to correspond to precisely 100 arcsec for all the fixed-grid values like search box size (60, 80, 100, ...).  Using 4.97... could be made to work too (maybe even right now), but not sure what it buys us.  Even the more exact value in the guide characteristics is an approximation since the plate scale varies with position.